daemon/oci_windows.go

```diff
@@ -183,9 +193,72 @@ func (daemon *Daemon) createSpec(c *container.Container) (*libcontainerd.Spec, e
 			Iops: &c.HostConfig.IOMaximumIOps,
 		},
 	}
+
+	// Add specific Windows credentials
+	if len(c.HostConfig.CredentialSpec) > 0 {
+		v := strings.ToLower(c.HostConfig.CredentialSpec)
+		f := strings.SplitAfter(v, "file://")
```

Comment:
Maybe this whole block would be simpler using `strings.HasPrefix` and `strings.TrimPrefix`? You wouldn't have to deal with the length of the resulting split string.